### ISSUE TITLE
Arrow shafts

### ISSUE THREAD

**Issue by bowiRS** [oldschoolgg/oldschoolbot#5643]
Arrow shafts can be bought in the Ranging Guild so it would be appropriate if they get added to /buy

**Comment by TURB042O** [comment]
> Arrow shafts can be bought in the Ranging Guild so it would be appropriate if they get added to /buy

I see your point. but then we'd have to add every item from every guild to </buy:0> and that would take the point of skilling out of the bot.

**Comment by bowiRS** [comment]
Correct, a LOT more items need to be added to buy, most notably high level runes. The fact that so many items are missing is no argument to avoid making a start to fixing it

**Comment by LuxrayElite** [comment]
Refer to Magna's comment on this pr : https://github.com/oldschoolgg/oldschoolbot/pull/3177

Short discussion about a similar thing in Devs: 
https://discord.com/channels/342983479501389826/648196527294251020/1087520937978642522

Tldr: Magna would want purchase limits imposed on items and some items maybe considered clutter (although not directly related to this issue)
